models.py

```diff
@@ -0,0 +1,35 @@
+import os
+import random
+orders={}
+class order:
+
+    def __init__(self):
+        self.orders=orders

```

Comment:
Using `randinit` doesnot guarantee you in any way that your `ids` are going to be unique , use the `uuid ` module if you intend to `generate unique ids`